templates/etherpad/meta.yaml

```diff
@@ -0,0 +1,39 @@
+name: Etherpad
+description: A modern really-real-time collaborative document editor.
+instructions: Check the environment tab for your API key
+changeLog:
+  - date: 2023-3-14
+    description: first release
+links:
+  - label: Website
+    url: https://etherpad.org
+  - label: Documentation
+    url: https://etherpad.org/doc/v1.8.12/
+  - label: Github
+    url: https://github.com/ether/etherpad-lite
+contributors:
+  - name: Supernova3339
+    url: https://github.com/Supernova3339
+schema:
+  type: object
+  required:
+    - projectName
+    - appServiceName
+    - appServiceImage
+    - databaseServiceName
+  properties:
+    projectName:
+      type: string
+      title: Project Name
+    appServiceName:
+      type: string
+      title: App Service Name
+      default: etherpad
+    appServiceImage:
+      type: string
+      title: App Service Image
+      default: platoniq/etherpad:1.8.12
```

Comment:
What are the benefits of using `platoniq/etherpad` instead of the official `etherpad/etherpad`?